### PR TITLE
Add CLI command for syncing media

### PR DESCRIPTION
**Pull request by tormjens**
Syncing lots of media to Cloudinary via the bulk action is less than optimal. For most setups it will time out if you try to add a large number of images. And boy does it take a long time to do it a few images at a time.

Since I am lazy and would rather write something that will do the job for me I wrote a WP CLI command for running the upload process via cli.

To start uploading all your media, you simply use (if you have WP CLI installed globally):
```
wp cloudinary-import
```

### CODE REVIEW

File: cloudinary-image-management-and-manipulation-in-the-cloud-cdn/lib/Cli.php [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            }
+        }
+
+        public function uploadAttachment($attachment)
+        {
+            return $this->plugin->upload_to_cloudinary($attachment->ID, true);
+        }
+
+    }
+}
```
Comment (QuentinMichinot) [inline cloudinary-image-management-and-manipulation-in-the-cloud-cdn/lib/Cli.php]: Forget ligne at the end of code ?!

**Comment by const-cloudinary** [comment]
Merged manually, see #27.

Thank you for contribution!

**Comment by stefanomonteiro** [comment]
Is there any command to migrate away?

**Comment by shirlymanor** [comment]
Yes, You can go to your media and choose to detach. You can do it for each image or by choosing all the images and choose to migrate away.

**Comment by stefanomonteiro** [comment]
Using a CLI command?

**Comment by shirlymanor** [comment]
Currently, we are not supporting CLI command. only from the UI.

**Pull request closed without merging**